service/resource/persistentvolume/error.go

```diff
@@ -0,0 +1,24 @@
+package persistentvolume
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = microerror.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
```

Comment:
Also comments. I think either all of these should have GoDoc or none of them should.